[2025-03-06T17:00:51.555+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-06T17:00:51.597+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: TD_pipeline_POC.merge_csv scheduled__2024-08-02T00:00:00+00:00 [queued]>
[2025-03-06T17:00:51.629+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: TD_pipeline_POC.merge_csv scheduled__2024-08-02T00:00:00+00:00 [queued]>
[2025-03-06T17:00:51.632+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-06T17:00:51.681+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): merge_csv> on 2024-08-02 00:00:00+00:00
[2025-03-06T17:00:51.700+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=19043) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-06T17:00:51.703+0000] {standard_task_runner.py:72} INFO - Started process 19063 to run task
[2025-03-06T17:00:51.711+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'TD_pipeline_POC', 'merge_csv', 'scheduled__2024-08-02T00:00:00+00:00', '--job-id', '585', '--raw', '--subdir', 'DAGS_FOLDER/dummyDag.py', '--cfg-path', '/tmp/tmpy0p_q6cz']
[2025-03-06T17:00:51.730+0000] {standard_task_runner.py:105} INFO - Job 585: Subtask merge_csv
[2025-03-06T17:00:51.902+0000] {task_command.py:467} INFO - Running <TaskInstance: TD_pipeline_POC.merge_csv scheduled__2024-08-02T00:00:00+00:00 [running]> on host c94cd9e82518
[2025-03-06T17:00:52.115+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='TD_pipeline_POC' AIRFLOW_CTX_TASK_ID='merge_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-08-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-02T00:00:00+00:00'
[2025-03-06T17:00:52.120+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-06T17:00:52.302+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-06T17:00:52.346+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-06T17:00:52.358+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=TD_pipeline_POC, task_id=merge_csv, run_id=scheduled__2024-08-02T00:00:00+00:00, execution_date=20240802T000000, start_date=20250306T170051, end_date=20250306T170052
[2025-03-06T17:00:52.693+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-06T17:00:52.764+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-06T17:00:52.780+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
